test/plugins/integrations/integrations.test.js

```diff
@@ -0,0 +1,31 @@
+const Thundra = require('../../../dist/index');
+process.env.AWS_LAMBDA_LOG_STREAM_NAME = '2018/03/02/[$LATEST]applicationId';
+
+describe('Integrations', () => {
```

Comment:
Do we have integration tests? If not, we can remove this test because it doesn't test the features.